src/main/java/use_cases/session_load/SessionLoadIB.java

```diff
@@ -0,0 +1,5 @@
+package use_cases.session_load;
+
+public interface SessionLoadIB {
```

Comment:
Missing documentation here :)

Reply:
Yeah, for some reason I was in the idea that we only needed JavaDocs for the concrete implementations and not the interfaces themselves, but I corrected it and added documentation to all of them now

Reply:
Sounds good! I think we will need to go over all documentation anyways so that everything looks coherent.